packages/glyphs-block/src/GlyphsBlock.tsx

```diff
@@ -40,7 +46,18 @@ export const GlyphsBlock = ({ appBridge }: BlockProps): ReactElement => {
                         outlineWidth: borderWidth,
                         outlineColor: toRgbaString(borderColor),
                     }),
-                    ...getRadiusStyles(radiusChoice, hasRadius, radiusValue),
+                    ...(isFirstChar && {
+                        borderTopLeftRadius: getRadiusValue(radiusChoice, hasRadius, radiusValue),
+                    }),
+                    ...(isSixthChar && {
+                        borderTopRightRadius: getRadiusValue(radiusChoice, hasRadius, radiusValue),
+                    }),
+                    ...(isBottomLeftChar && {
+                        borderBottomLeftRadius: getRadiusValue(radiusChoice, hasRadius, radiusValue),
+                    }),
+                    ...(isBottomRightChar && {
+                        borderBottomRightRadius: getRadiusValue(radiusChoice, hasRadius, radiusValue),
+                    }),
```

Comment:
this now gets a bit big :D. you could use `Object.assign()`. Wdyt?

Like: 
const style = Object.assign(
            {},
            hasBackground && { backgroundColor: toRgbaString(backgroundColor) },
...
}

Reply:
I think it's easier understandable without Object.assign(), but moved the styles to a const to make it a bit cleaner.